content/en/mobile/datadog_for_intune.md

```diff
@@ -65,6 +65,10 @@ Permissions are automatically added when configuring the application:
 | Microsoft Graph                         | `User.Read`                              | Sign in and read user profile                   | Delegated |
 | Microsoft Mobile Application Management | `DeviceManagementManagedApps.ReadWrite`  | Read and Write the User's App Management device | Delegated |
 
+**Note:** Only those two permissions are used by the mobile-app. When granting consent you may see more permissions, this is due to the mobile-app sharing the same Microsoft Entra application than our web-app and the [Microsoft Teams Integration](https://docs.datadoghq.com/integrations/microsoft_teams). If you're not using it you can revoke those permissions from the **Permissions** tab on the page detailed in _step 3_.
+
+**Note 2:** While `DeviceManagementManagedApps.Read` exists, this is an MS Graph API permission and the mobile-app requires a MAM permission. `DeviceManagementManagedApps.ReadWrite` is the only MAM one available but **mobile-app doesn't write anything** and only use read in that case.
```

Comment:
```suggestion
```suggestion
- While `DeviceManagementManagedApps.Read` exists, it is an MS Graph API permission, and the mobile app requires a MAM permission. `DeviceManagementManagedApps.ReadWrite` is the only MAM permission available, but the mobile app only uses the write permission and doesn't write anything.
```
```